### ISSUE TITLE
Symbols prefacing lists are created with css :before

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon#3628]
## Environment

> Operating system
Windows 7

> Browser
Chrome/Firefox

## Detailed description
The symbols used for ordered (numbers and letters) and unordered (dashes and bullets) lists appear to be placed before the text using CSS ::before
There are some potential issues with this, including the inability to copy the list indicators.
While this may be a technical failure of 1.3.1, I have confirmed that the list indicators are announced by JAWS and are also visible in High Contrast mode.
So I do not regard this as a high priority issue, but it does appear to be a technical failure of WCAG (I am including the specific failure technique).

> What version of the Carbon Design System are you using?
v10, storybook, themes

> What WCAG 2.1 checkpoint does the issue violate?
[1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) See:
[Failure of Success Criterion 1.3.1 due to inserting non-decorative content by using :before and :after pseudo-elements and the 'content' property in CSS](https://www.w3.org/WAI/WCAG21/Techniques/failures/F87)

**Comment by dakahn** [comment]
Since this isn't a defect and is low priority we're going to close. Looks like it's a potential violation but through @snidersd 's testing no information is lost. Happy to reopen if we need to though.
